Source/Core/VideoCommon/FramebufferManager.cpp

```diff
@@ -508,6 +508,9 @@ void FramebufferManager::RefreshPeekCache()
     }
   }
 
+  m_efb_depth_cache.valid = true;
```

Comment:
Shouldn't these be set with `PopulateEFBCache` checks above if needed?

Reply:
PopulateEFBCache is only called if we have tiles that were used recently,

So it's possible that we don't have any tiles that were recently used in which case it's never called.

Reply:
Ah I see.  So maybe it would be more appropriate for this to go outside the `for` loop above?  The else statement is already handled.